src/cmds/pbsdsh.c

```diff
@@ -226,6 +329,14 @@ main(int argc, char *argv[], char *envp[])
 					err = 1;
 				}
 				break;
+			case 'h':
+				onenode=0;
+				targethost=optarg;		
+				onenode=find_hostline(targethost);
```

Comment:
Do you need to set onenode = 0 above?  You are just overwriting it here.  Also put spaces around your equals.